tests/ui_tests/common.py

```diff
@@ -99,13 +99,13 @@ def screens_and_hashes(screen_path: Path) -> tuple[list[Path], list[str]]:
     if not screen_path.exists():
         return [], []
 
-    hashes = []
-    paths = []
+    hashes: list[str] = []
+    paths: list[Path] = []
     for file in sorted(screen_path.iterdir()):
         paths.append(file)
         if len(file.stem) == 32:
             try:
-                hashes.append(bytes.fromhex(file.stem))
+                hashes.append(str(bytes.fromhex(file.stem)))
```

Comment:
you're making str from bytes from str :)
more generally though, I think `if len(file.stem)` branch is a leftover, maybe it should go away completely?

Reply:
Yes, why deleting one operation when you can add a new operation :D

`if len(file.stem) == 32` would be a nice shortcut if it would be used over `screens` dir, but in `actual` and `recorded` it does not make much sense, as the names are `00000001.png` etc.

Whole `if` branch deleted in [78785b0](https://github.com/trezor/trezor-firmware/pull/2750/commits/78785b0f6d9a163e752675f37341ae75cde08280)